### ISSUE TITLE
Evaluating issue

### ISSUE THREAD

**Issue by jakublipinski** [kwotsin/Tensorflow-Xception#6]
Hi Kwot! Thanks for the Tensorflow Xception.

I run the `1789` steps of the `pythong train_flowers.py` on my computer (CPU) and according to the Tensorboard I reached `accurrancy=54%`. However after I launch `python eval_flowers.py` I get the folllowing output and the model predicts every eval pitcure as `dandelion` (I modified the code to print the text label instead of displaying it on the picture)  . What am I doing wrong?
```
python eval_flowers.py 
INFO:tensorflow:Restoring parameters from ./log/model.ckpt-1772
INFO:tensorflow:global_step/sec: 0
INFO:tensorflow:Epoch: 1/1
INFO:tensorflow:Current Streaming Accuracy: 0.0000
INFO:tensorflow:Global Step 1: Streaming Accuracy: 0.0000 (77.42 sec/step)
INFO:tensorflow:Global Step 2: Streaming Accuracy: 0.1389 (23.17 sec/step)
INFO:tensorflow:Global Step 3: Streaming Accuracy: 0.1528 (18.02 sec/step)
INFO:tensorflow:global_step/sec: 0.033286
INFO:tensorflow:Global Step 4: Streaming Accuracy: 0.1667 (16.17 sec/step)
(...)
INFO:tensorflow:Global Step 10: Streaming Accuracy: 0.2469 (15.40 sec/step)
INFO:tensorflow:global_step/sec: 0.0584161
INFO:tensorflow:Global Step 11: Streaming Accuracy: 0.2472 (15.29 sec/step)
(...)
INFO:tensorflow:Global Step 18: Streaming Accuracy: 0.2484 (14.36 sec/step)
INFO:tensorflow:global_step/sec: 0.0666661
INFO:tensorflow:Global Step 19: Streaming Accuracy: 0.2454 (16.38 sec/step)
(...)
INFO:tensorflow:Global Step 26: Streaming Accuracy: 0.2367 (14.29 sec/step)
INFO:tensorflow:global_step/sec: 0.0666666
INFO:tensorflow:Global Step 27: Streaming Accuracy: 0.2372 (14.73 sec/step)
INFO:tensorflow:Global Step 28: Streaming Accuracy: 0.2428 (14.41 sec/step)
INFO:tensorflow:Global Step 29: Streaming Accuracy: 0.2460 (14.71 sec/step)
INFO:tensorflow:Global Step 30: Streaming Accuracy: 0.2462 (14.35 sec/step)
INFO:tensorflow:Final Streaming Accuracy: 0.2444
INFO:tensorflow:Prediction: dandelion 
 Ground Truth: dandelion 
 Probability: 0.30221
INFO:tensorflow:Prediction: dandelion 
 Ground Truth: dandelion 
 Probability: 0.302133
INFO:tensorflow:Prediction: dandelion 
 Ground Truth: dandelion 
 Probability: 0.302201
INFO:tensorflow:Prediction: dandelion 
 Ground Truth: tulips 
 Probability: 0.302064
INFO:tensorflow:Prediction: dandelion 
 Ground Truth: daisy 
 Probability: 0.302107
INFO:tensorflow:Prediction: dandelion 
 Ground Truth: roses 
 Probability: 0.30195
INFO:tensorflow:Prediction: dandelion 
 Ground Truth: roses 
 Probability: 0.302043
INFO:tensorflow:Prediction: dandelion 
 Ground Truth: dandelion 
 Probability: 0.3023
INFO:tensorflow:Prediction: dandelion 
 Ground Truth: tulips 
 Probability: 0.302177
INFO:tensorflow:Prediction: dandelion 
 Ground Truth: daisy 
 Probability: 0.301616
INFO:tensorflow:Model evaluation has completed! Visit TensorBoard for more information regarding your evaluation.
```

**Comment by kwotsin** [comment]
It seems that your model isn't as accurate as it is. The evaluation accuracy is just 24%, which means it is likely the model doesn't really predict well. You could try experimenting with the hyperparameters to improve your model or train for a longer time.
